.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"

```

Comment:
Would you mind updating these intervals to be less frequent to reduce PR noise in the event of frequent dependency updates. Perhaps weekly or every two weeks?

Reply:
It's now updated to run on a weekly schedule. 
See e0f1b7b :-)